pytorch_lightning/loggers/wandb.py

```diff
@@ -22,16 +22,18 @@
 
 import torch.nn as nn
 
+from pytorch_lightning.loggers.base import LightningLoggerBase, rank_zero_experiment
+from pytorch_lightning.utilities import rank_zero_only, _module_available
+from pytorch_lightning.utilities.warning_utils import WarningCache
+
+_WANDB_AVAILABLE = _module_available("wandb")
```

Comment:
_WANDB_RUN_AVAILABLE = _module_available("wandb.wandb_run.Run")

Reply:
do we need a special for `Run`? is there a case when wandb does not contain Run?

Reply:
We could do.